src/components/OptInFeatures/index.tsx

```diff
@@ -0,0 +1,176 @@
+    borderTop: isMobile ? (isDisclosure ? '1px solid #b86f6f' : '1px solid #ddd') : 'none',
+    fontSize: '12px',
+    color: isDisclosure ? '#b86f6f' : 'inherit',
+  });
+
+  const background = {
+    background:
+      'url(/assets/trees-right.png) no-repeat 100% 100%/auto 248px, url(/assets/trees-left.png) no-repeat 0 100%/auto 175px, url(/assets/water.png) repeat-x 0 100%/auto 142px, url(/assets/mountain.png) no-repeat 0 100%/auto 233px, url(/assets/far-mountain.png) no-repeat 100% 100%/auto 317px, url(/assets/background.png) no-repeat 100% 0/90% 633px, linear-gradient(to bottom right, rgb(255, 255, 255) 0%, rgb(199, 226, 234) 100%) no-repeat 0 0/auto',
+  };
+
+  return (
+    <Box
+      sx={{
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        fontFamily: 'monospace',
+        padding: '40px',
+        ...background,
+      }}
+    >
+      <PageSnackbar />
+      {preferences === undefined ? (
+        <LinearProgress color='success' />
+      ) : (
+        <Grid item xs={12}>
+          <Box
+            sx={{
+              display: 'flex',
+              color: '#444',
+              fontWeight: 'bold',
+              fontSize: '18px',
+              marginBottom: '30px',
+              justifyContent: 'center',
+            }}
+          >
+            Opt-in to see experimental or work-in-progress features
+          </Box>
+          {OPT_IN_FEATURES.map((f, i) => (
+            <Stack
+              spacing={2}
+              sx={{
+                border: '1px solid #ddd',
+                padding: '10px',
+                borderRadius: '5px',
+                backgroundColor: 'rgba(255,255,255,0.8)',
+                marginBottom: '10px',
+              }}
+              key={i}
+            >
+              <Grid
+                item
+                xs={12}
+                sx={{ display: 'flex', flexDirection: isMobile ? 'column' : 'row', alignItems: 'center' }}
+              >
+                <Grid item xs={gridSize()}>
+                  <Box sx={{ display: 'inline-block', color: '#444', fontSize: '14px', whiteSpace: 'pre' }}>
+                    <Switch
+                      checked={preferences[f.preferenceName] === true}
+                      onChange={(event) => savePreference(f, event.target.checked)}
+                    />
+                    &nbsp;{f.name}
+                  </Box>
+                </Grid>
+                <Grid item xs={gridSize()} sx={gridStyle(false)}>
+                  {f.description.map((d, index) => (
+                    <Box key={index} sx={{ marginBottom: '5px' }}>
+                      {d}
+                    </Box>
+                  ))}
+                </Grid>
+                <Grid item xs={gridSize()} sx={gridStyle(true)}>
+                  {f.disclosure.map((d, index) => (
+                    <Box key={index} sx={{ marginBottom: '5px', fontStyle: 'italic' }}>
```

Comment:
All the styles here are up to us - this is not end-customer facing UI. Devs can feel free to change styles or experiment with new css/widgets any time.